app/client/lib/highlight.js

```diff
@@ -1,5 +1,4 @@
 var find = require('findandreplacedomtext')
-var css = require('dom-css')
 
 // Highlights all the words in text inside a node
 function Highlight (node, text, wrapper) {
```

Comment:
Also `text` can just be `*`, indicating that the user is showing their entire local collection.

So the first check should be something like:

```js
if (!text || text.trim() === '*') return
```